Documentation/community_features.md

```diff
@@ -57,6 +57,9 @@ Synchronization modes accessible through the "LFO SYNC" shortcut.
 ### Kit Keyboard View
  - ([#112]) All-new use for the "keyboard" button in kit clips, uses the main pad grid for MPC-style 16 level playing. Horizonatal encoder scrolls by one pad at a time, allowing positioning drums left to right, and vertical encoder jumps vertically by rows.
 
+### Quantize and Humanize Notes
```

Comment:
The headers are intended to be the view where this is possible, so maybe "All Clip Views"?

Reply:
Updated with that in mind.